routes/api.php

```diff
@@ -17,3 +19,23 @@
 Route::middleware('auth:sanctum')->get('/user', function (Request $request) {
     return $request->user();
 });
+
+Route::group([
+    'prefix' => 'v1/cities',
+    'controller' => CityController::class
+], static function () {
+    Route::get('/', 'index');
+    Route::post('/', 'store');
+    Route::put('/{city}', 'update');
+    Route::delete('/{city}', 'destroy');
+});
+
+Route::group([
+    'prefix' => 'v1/airlines',
+    'controller' => AirlineController::class
+], static function () {
+    Route::get('/', 'index');
+    Route::post('/', 'store');
+    Route::put('/{airline}', 'update');
+    Route::delete('/{airline}', 'destroy');
+});
```

Comment:
When working with API routes, it's ideal for the controller to be API-specific. I'm telling you this because both the `CityController` and `AirlineController` have a method named `indexView` that returns a view, which isn't an expected behavior for an API. You could create an `Api` directory inside of `Controllers`, create both of these controllers and move all these methods there.